package.json

```diff
@@ -47,7 +47,7 @@
         "@types/minimatch": "latest",
         "@types/minimist": "latest",
         "@types/mkdirp": "latest",
-        "@types/mocha": "latest",
```

Comment:
any specific reason for this change?

Reply:
Nope, I don't know where that came from.